src/components/Search.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
```

Comment:
You have 4 spaces of indentation here, theres a bit of discrepancy as some only have 2.
Theres no hard and fast rule about how many spaces you should use but its important to be consistent whichever you choose